harness/determined/exec/prep_container.py

```diff
@@ -61,6 +61,11 @@ def trial_prep(info: det.ClusterInfo, cert: certs.Cert) -> None:
         model_def.extractall(path=constants.MANAGED_TRAINING_MODEL_COPY)
         model_def.extractall(path=".")
 
+    # pre-0.18.2 code wrote tensorboard stuff under /tmp/tensorboard
```

Comment:
I think this will be pre 0.18.3 code, right?  This PR won't land in the 0.18.2 release.

There may be other instances of "0.18.2" in the content of this pr, come to think of it.